code/datums/spells/cluwne.dm

```diff
@@ -0,0 +1,88 @@
+/obj/effect/proc_holder/spell/targeted/touch/cluwne
+	name = "Curse of the Cluwne"
+	desc = "Turns the target into a fat and cursed monstrosity of a clown."
+	hand_path = /obj/item/weapon/melee/touch_attack/cluwne
+
+	school = "transmutation"
+
+	charge_max = 600
+	clothes_req = 1
+	cooldown_min = 200 //100 deciseconds reduction per rank
+
+	action_icon_state = "clown"
+
+/mob/living/carbon/human/proc/makeCluwne()
+	to_chat(src, "<span class='danger'>You feel funny.</span>")
+	adjustBrainLoss(80)
+	nutrition = 9000
+	overeatduration = 9000
+	confused = 120
```

Comment:
On Goon, cluwning effectively increases the chance of you stepping on the wrong tile by 66%, permanently [[source]](https://github.com/goonstation/goonstation-2016/blob/master/code/datums/abilities/wizard/cluwne.dm#L47). 4 minutes of confusion here will suffice, as there isn't a permanent alternative that's not ridiculously debilitating.